src/main/java/core/basesyntax/Colors.java

```diff
@@ -0,0 +1,12 @@
+package core.basesyntax;
+
+public enum Colors {

```

Comment:
enum name should be in singular, for more  read convention : https://google.github.io/styleguide/javaguide.html#s4.8.1-enum-classes